web-jehlomat/src/screens/Nalezy/Nalezy.tsx

```diff
@@ -54,36 +55,14 @@ const Nalezy: FunctionComponent = () => {
 
         load()
             .then(data => {
+                // setLoader({ resp: mock });
```

Comment:
same, can we remove this commented code? If not,  some explanation would be nice ....

Reply:
I'm using it for testing purposes (not able to create any record at least at this moment), but sure, I can put there an explanation)